packages/runtime/container-runtime/README.md

```diff
@@ -51,6 +51,52 @@ issue](https://github.com/microsoft/FluidFramework/wiki/Submitting-Bugs-and-Feat
+
+### Motivation for Data Virtualization
+
+This section talks about how the system used to work before Data virtualization. Fluid (through its APIs) does expose
+some levels of data virtualization. But under covers, there is almost none. The whole content of the file is downloaded
+in one go. Due to limitation of data virtualization, FF holds ALL blobs in snapshot as those might be required in the
+future. Any delayed loading (through FF APIs) results in loading state of DDSs at a sequence number of snapshot we
+booted from, and application of all the ops up till current sequence number. While application may choose not to load
+some data stores immediately on boot (and realize some saving in time and memory by not allocating appropriate app
+state for such datastores), FF still pays the costs for such content. It also continues to pay the cost for all such
+content indefinitely, even if those datastores were loaded (this could theoretically be optimized, but now there is
+no good way for driver to learn what blobs are safe to discard due to limitations in interface and lack of appropriate
+semantics at driver API layer).
+
+### Improvement with Data Virtualization
+
+With this, we will provide a capability to:
+
+- Exclude some sub-trees from snapshot payload, thus allowing faster transfer times / boot times and smaller initial
+memory footprint.
+- Ability to delay-load data stores later.
+
+Container Runtime Apis like IContainerRuntimeBase.createDataStore and IContainerRuntimeBase.createDetachedDataStore
+provides an argument `loadingGroupId` which allows apps to mark a datastore at time of creation currently. Every data
+store is assigned a groupID. Not providing groupID (on API) means that default ID is used. This groupId represents the
+group of the datastore within a container or its snapshot. When a container is loaded initially, only datastores which
+belongs to `default` group are fetched from service and can be loaded on demand when requested by user. This decreases
+the amount of data which needs to be fetched during load and hence provides faster boot times for the container.
+Attempting to load any datastore within a non-default group results in fetching all content/datastores marked with same
+groupId and thus helps faster realization/loading of other datastores within that group faster.
+So, one network will be required to fetch content for a group when a datastore from a group is requested by an
+application.
+So to summarize, when datastore is assigned to a group, content of such data store will not be loaded with initial load
+of container. It will be loaded only when any datastore with such groupID is realized.
+This will improve the boot perf. Data virtualization or providing the `loadingGroupId` will however decrease the
+performance of loading of those datastores while improving the overall performance of container load. However,
```

Comment:
what does this mean?

Reply:
It means that DV will improve the boot perf of initial load of container but decrease the performance of loading of datastores with groupid as it will now involve making a network call while before no network call was required as the snapshot already existed in memory.

Reply:
I could try to make it clearer here in readme.